app/controllers/iss_alerts_controller.rb

```diff
@@ -2,16 +2,27 @@ class IssAlertsController < ApplicationController
   def create
     phone_number = '+1' + params['phone']
     formatted_phone_number = params['phone'][0..2] + '-' + params['phone'][3..5] + '-' + params['phone'][6..9]
-    scheduled_time = Time.parse(params['scheduled_time'])
+    location = formatted_location(params['location'])
+    time_zone = Timezone.lookup(location[1], location[0]).name
```

Comment:
This probably should have had a comment in the code, but the location comes from the facade in long/lat and the API require lat/long